src/tools/genpolicy/genpolicy-settings.json

```diff
@@ -203,6 +203,7 @@
     ],
     "common": {
         "cpath": "/run/kata-containers/shared/containers",
+        "mount_source_cpath": "/run/kata-containers/shared/containers",
```

Comment:
Hi @Redent0r !

I'm having a hard time to understand this change on genpolicy code, and why it's necessary now and for TDX. Could you please make it into a separated commit, and also write a message explaining what/why/how of the change?

Reply:
Added this change on a separate commit with clear explanation :) Thanks!